src/controllers/DrawingModeController.js

```diff
@@ -46,8 +46,8 @@ class DrawingModeController extends AnnotationModeController {
         // light, dark, or no value given), then we want to use our draw
         // header. Otherwise we expect header UI to be handled by Preview’s
         // consumer
-        if (data.options.header !== 'none') {
-            this.setupHeader(this.container, shell);
+        if (data.options.header !== 'none' || this.headerElement) {
+            this.setupHeader(this.headerElement, shell);
```

Comment:
how would this work in the cases where data.options.header !== 'none' and this.headerElement is null, would passsing in this.headerElement to the setupHeader method work?

Reply:
The check in `Annotator.js` on line 117 is the attempt to handle this. If box content preview header is not none, and this.headerElement is null, then we fallback to this.container